internals/search.py

```diff
@@ -319,9 +319,14 @@ def make_cache_key(
   """Return a redis key string to store cached search results."""
   return '|'.join([
       FeatureEntry.SEARCH_CACHE_KEY,
-      user_query, str(sort_spec), str(show_unlisted),
-      str(show_deleted), str(show_enterprise), str(start), str(num),
-      str(name_only),
+      user_query,
+      'sort_spec=' + str(sort_spec),
+      'show_unlisted=' + str(show_unlisted),
+      'show_deleted=' + str(show_deleted),
+      'show_enterprise=' + str(show_enterprise),
+      'start=' + str(start),
+      'num=' + str(num),
+      'name_only=' + str(name_only),
```

Comment:
Note: we only cache name_only queries for now. This part of the key is optional

Reply:
I think it is best to include it, just in case we decide to broaden what we cache.